### PR TITLE
improvement added

### PR DESCRIPTION
**Pull request by kdiffin**

### CODE REVIEW

File: src/components/Calculator.tsx [hunk 1]
```diff
@@ -46,7 +48,9 @@ function Calculator() {
 
         ]))
     },[calc])
-    function Zesty(){
+
+    //changed function name to actually mean something
+    function hideCalculated(){
```
Comment (RMuradli) [inline src/components/Calculator.tsx:53]: Nice job! Now I can review and comment on the files and lines that you changed.  
Later I will create a new branch and put my comments within the code and open a pr so you can see them and improve accordingly.

File: src/components/Calculator.tsx [hunk 2]
```diff
@@ -166,7 +170,7 @@ function Calculator() {
             
             <button   
                 className='button operators' 
-                onClick={function(event){ calculate(); Zesty()}}
+                onClick={function(event){ calculate(); hideCalculated()}}
```
Comment (RMuradli) [inline src/components/Calculator.tsx:173]: This review and comments should not stop you from your current project. You continue your work, and when you have time and energy you come back, and see if you want to fix those comments or rather decide to continue as it is.

Comment (RMuradli) [inline src/components/Calculator.tsx:173]: Overall you have done a great job! The code is nice and clean, works properly.

Just remember, try to always break apart your project into pieces like a puzzle, once a working piece is ready(like a complex function, or a small component), commit it with a message that would help you track it if you need. And always create branch before coding, with a purpose, so you would know that the branch you opened is targeted to develop certain feature, and you wont mixup other stuff into it (for example, functions of calculator operations, you develop them, test them in console, and merge them to master, and delete the branch. This way you'll know that functions are ready, and in the next branch, you only have to use them in your components). And always name those branches for your purposes, like feature(if new feature code is written, improvement to fix comments, bugfix to fix bug, etc)

Comment (RMuradli) [reply src/components/Calculator.tsx:173]: Later I will leave you with advice within the files, so you know how you can optimize your current and other projects further.

Comment (kdiffin) [reply src/components/Calculator.tsx:173]: alright thank you so much for the review rashad i will proceed to merge it to my master branch

**Pull request merged**